src/gymnasium_interface/Pycram_gym_env.py

```diff
@@ -0,0 +1,75 @@
+import gymnasium as gym
+from gymnasium.spaces import Discrete
+from .task_executor import PyCRAMTaskExecutor
+from pycram.process_module import simulated_robot
+import sys
+
+class PyCRAMGymEnv(gym.Env):
+    def __init__(self, actions, default_params=None, objects=None, reward_function=None):
+        """
+        Initializes the Gymnasium environment.
+
+        Args:
+            actions (list): List of valid action names (e.g., ["navigate", "pick_up", "place"]).
```

Comment:
Please stick to the rst style doc strings, look at the other methods and classes for reference